src/main/java/software/amazon/awssdk/crt/s3/ChecksumAlgorithm.java

```diff
@@ -0,0 +1,48 @@
+package software.amazon.awssdk.crt.s3;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ChecksumAlgorithm {
+
+    NONE(0),
+
+    CRC32_C(1),
```

Comment:
trivial: I usually see this written as "32C" without the underscore.

Reply:
I just checked the S3 model, it actually uses `CRC32C`, not sure why Java SDK codegen adds underscore 🤷🏼‍♀️  , so it might be better to remove the underscore here, but either way is fine with me.

```
public enum ChecksumAlgorithm {
    CRC32("CRC32"),

    CRC32_C("CRC32C"),

    SHA1("SHA1"),

    SHA256("SHA256")
```